receiver/hostmetricsreceiver/go.sum

```diff
@@ -1,277 +0,0 @@
-dario.cat/mergo v1.0.0 h1:AGCNq9Evsj31mOgNPcLyXc+4PNABt905YmuqPYYpBWk=
```

Comment:
Diffs do not look correct

Reply:
good catch...

Reply:
i've re-triggered renovate

Reply:
still the same, I guess this branch needs a manual `make gotidy`